packages/auto-label/src/auto-label.ts

```diff
@@ -31,8 +52,7 @@ const DEFAULT_CONFIGS = {
   },
 };
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const colorsData = require('./colors.json');
+import colorsData from './colors.json';
```

Comment:
Move to be next to the other imports at the top? Or is that not a thing?

Reply:
We could, just didn't move it since it was already here